debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/metrics/MongoDbSnapshotChangeEventSourceMetrics.java

```diff
@@ -27,11 +27,9 @@ public class MongoDbSnapshotChangeEventSourceMetrics extends DefaultSnapshotChan
     private AtomicLong numberOfDisconnects = new AtomicLong();
 
     public <T extends CdcSourceTaskContext> MongoDbSnapshotChangeEventSourceMetrics(T taskContext, ChangeEventQueueMetrics changeEventQueueMetrics,
-                                                                                    EventMetadataProvider metadataProvider, int maxTasks) {
+                                                                                    EventMetadataProvider metadataProvider) {
         super(taskContext, changeEventQueueMetrics, metadataProvider,
-                maxTasks > 1
-                        ? Collect.linkMapOf("context", "snapshot", "server", taskContext.getConnectorName(), "task", taskContext.getTaskId())
-                        : Collect.linkMapOf("context", "snapshot", "server", taskContext.getConnectorName()));
+                Collect.linkMapOf("context", "snapshot", "server", taskContext.getConnectorName(), "task", taskContext.getTaskId()));
```

Comment:
Hi, @Naros 

thanks for tackling this issue.

I'm concerned that I don't see some changes related with task id initialization.
From my perspective, `taskContext.getTaskId()` always returns `0` because `MongoDbTaskContext` class calls superclass constructor without specifying the task id: 
```
public MongoDbTaskContext(Configuration config) {
    super(Module.contextName(), config.getString(MongoDbConnectorConfig.LOGICAL_NAME), Collections::emptySet);
```
```
public CdcSourceTaskContext(String connectorType, String connectorName, Supplier<Collection<? extends DataCollectionId>> collectionsSupplier) {
    this(connectorType, connectorName, "0", collectionsSupplier);
}
```
`MongoDbConnectorConfig.TASK_ID` can be used to get the task id from configuration.